packages/strapi-plugin-versioning/src/services/Versioning.ts

```diff
@@ -0,0 +1,47 @@
+import { Context } from "koa";
+
+type Entity = any;
+type Data = Array<Entity>;
+
+const isModelExists = (ctx: Context): boolean =>
+  (ctx.params !== undefined && ctx.params.model !== undefined) ||
+  Object.values(global.strapi.models).find(
+    (el) => el.collectionName == ctx.url.split("/")[1],
```

Comment:
```ctx.url.split("/")[1]```
is there no better way of doing it? I didn't notice in strapi core that they are taking collection name just form URL? 🤔 

(and use three ===)

Reply:
it is handling api from outside, and as I remember in api core it is handled in similar way.

Reply:
I search in repo strapi with version 3.0.6 and nothing found, it's not good idea to split url and take second element - theoretically it coulb be undefined